budgetportal/fixtures/provincial-infrastructure-projects.json

```diff
@@ -1,4 +1,12 @@
 [
+{
```

Comment:
Sorry, I forgot these will rely on financial years being defined. 

We can't just add a financial year to the provincial-infrastructre-project fixture. This fixture assumes you've run the financial years fixture.

Please create a fixture specifically for ProvInfraProjectContentTestCase named test_prov_infra_project_content with just the project and financial year needed for that case and remove the financial year here.

Reply:
new fixture added

https://github.com/vulekamali/datamanager/pull/190/commits/ba4c0bab3b384228c87e0c179a168c660de41480